contracts/strategies/AaveStrategyAvaxV1.sol

```diff
@@ -46,8 +46,7 @@ contract AaveStrategyAvaxV1 is YakStrategyV2Payable, ReentrancyGuard {
         rewardController = IAaveIncentivesController(_rewardController);
         tokenDelegator = ILendingPool(_tokenDelegator);
         rewardToken = IERC20(address(WAVAX));
```

Comment:
I believe base class also want `depositToken`

Reply:
no need, on the avax strategy depositToken is 0x0